### DISCUSSION TITLE
Add-on Store - merge all of users additions into "Your custom repositories" section

### DISCUSSION THREAD

**Discussion by SeanPM5** [home-assistant/frontend#11260] [Other feature requests]
## The request

Companion to https://github.com/home-assistant/frontend/issues/5732 kinda. With the repository management features getting moved to overflow menu, this add-on store page looks significantly cleaner now. Well done Ludeeus 😃 

There's one final part I feel can be cleaned up in a similar way:

![image](https://i.imgur.com/7eaTN7o.jpg)

This pic is actually after some very welcomed cleanups (was a lot uglier before with URL's and emails etc). But I think it would be even nicer still if these could all be merged together? There's two reasons for this:

1. Most repositories have no more than 1 or 2 add-ons, so there's a ton of wasted space (indicated by my red overlay). You often have a single card taking up an entire row with lots of white space after it.
2. The headings are also a waste of space, as they contain zero useful information. Worse yet, most custom repos use the "Hass.io" branding which has been replaced... So it could be confusing too.

So the idea is to merge these all into one section called "Your custom repositories" (or perhaps "Your added repositories"). Here's a very rough mockup.

![image](https://i.imgur.com/8JVIIQD.jpg)

I think it will look a lot nicer this way, far less wasted space and much more professional looking. 

And although you can't see it in my rushed mockup, a nice bonus is that since it's sorted alphabetically, Frenck's "Home Assistant Community Add-ons" (which is the largest collection of add-ons and has nice looking images and such), will always be the second section of the store right after the official add-ons. Hierarchy would always be guaranteed as Official addons > Community addons > Your addons.

**Comment by ludeeus** [comment]
Since no "The alternatives" where offered, I'd like to add one to this since it touches the same space.
With the introduction of the new layout for the integrations page, I feel like the current addon layouts are a bit "old", and it does not feel like a coherence experience.

As an alternative solution, I would like to purpose that with move the layout of the addons to be more like integrations.
![image](https://user-images.githubusercontent.com/15093472/81658824-6bbfcc80-9439-11ea-9267-24fcdf45359a.png)

Where you see the installed as cards like that, and for the "store" we implement something like the way you add new integrations (click the +, then search/scroll the list)

Maybe add a badge (chip) with the repo name?

**Comment by frenck** [comment]
Please note that multiple repositories can deliver the same add-on.

**Comment by ludeeus** [comment]
If installed those would be 2 different cards (with my example), and for the add list I was thinking of using the same list-style as the repo manager to show a little more information in that list (maybe with a pop-up of some sort to show the info? 🤷 )

**Comment by frenck** [comment]
Extreme, not normal user, example: I have all of the community add-ons 4 times. Local copies, stable, beta & edge repository. Every core add-on 2 times (local & stable).

While I do not consider that a normal usage, people are using things like the edge repository a lot, or making local copies of add-ons to customize.

**Comment by ludeeus** [comment]
You are a perfect example :)
If we do something that works for you and a default new setup, we have something.
It should not be changed until we can find something that works for both those scenarios.

**Comment by ludeeus** [comment]
How is this as an alternative?
![image](https://user-images.githubusercontent.com/15093472/94127390-284e8380-fe59-11ea-9beb-ef0b508655f6.png)

The "from" part would only show if you have multiple addons with the same name

**Comment by SeanPM5** [comment]
I like that idea.
